Signal/src/ViewControllers/InviteFlow.swift

```diff
@@ -88,17 +92,16 @@ class InviteFlow: NSObject, MFMessageComposeViewControllerDelegate, MFMailCompos
     func contactsPicker(_: ContactsPicker, didSelectMultipleContacts contacts: [Contact]) {
         Logger.debug("\(TAG) didSelectContacts:\(contacts)")
 
-        self.presentingViewController.dismiss(animated: true)
```

Comment:
`sendSMSTo` and `sendMailTo` already dismiss the contact picker; we don't want to double-dismiss.